test/lib/redis-store-spec.js

```diff
@@ -463,3 +466,39 @@ describe('defaults', function () {
   });
 });
 
+describe('wrap function', function () {
+
+  // Simulate retrieving a user from a database
+  function getUser(id, cb) {
+    setTimeout(function () {
+      cb(null, {id: id, name: 'Bob'});
+    }, 100);
+  }
+
+  var userId = 123;
+  var key = 'user_' + userId;
+  var fromCache = true;
+
+  it('should work with wrap function', function (done) {
```

Comment:
I am not sure this test is actually testing anything relevant. It should at least use the getUser to get something from a (redis store.